packages/suite/src/support/messages.ts

```diff
@@ -8625,7 +8625,7 @@ const messages = defineMessagesWithTypeCheck({
     TR_STAKE_STAKING_IS: {
         id: 'TR_STAKE_STAKING_IS',
         defaultMessage:
-            "Staking involves temporarily locking your Ethereum assets to support the blockchain's operation. In return, you'll earn additional Ethereum as a reward.",
+            "Staking involves temporarily locking your {network} assets to support the blockchain's operation. In return, you'll earn additional {network} as a reward.",
```

Comment:
Do we want to say @jirih-stsh 
- `earn additional ETH as a reward.` or `earn additional Ethereum as a reward.`
- `earn additional SOL as a reward.` or `earn additional Solana as a reward.`

Reply:
@tomasklim I would go for
- earn additional ETH as a reward.
- earn additional SOL as a reward.

Reply:
oki